swagger.yaml

```diff
@@ -0,0 +1,91 @@
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: My Tutorial project!!!
+host: localhost:8000
+tags:
+- description: API for design tutorial
+schemes:
+- http
+consumes:
+- application/json
+produces:
+- application/json
+paths:
+  "/Tutorial":
```

Comment:
1. URL segment should be always in lower case. 
2. No need to wrap string without spaces with quotes in YAML. Please do the same in the whole file.
```
paths:
  /tutorial:
     parameters:
```